### ISSUE TITLE
What does "group" mean ?

### ISSUE THREAD

**Issue by netbean2** [escribis/CyberResidencesOCL#3]
Everything is in the title, the group notion is not inside the model, so how do we understand it ?

For instance on constraint :  tutorInSameGroup
Tutors and tutored in one group.

**Comment by xcriptus** [comment]
You are right. The notion of "group" is not in the model. It has appeared after the initial design with the first client. It is actually useful but for now for have decide not to change the model to avoid major impacts in existing databases. 

In fact a group is corresponding to one tenant and to all residents that occupy a room rented by that tenant. For instance if ioannis is renting room r310, r322 and r325, all residents in these rooms are part of the ioannis' group.
